src/Backend/Facebook.php

```diff
@@ -26,8 +26,7 @@ public function getRequest($url)
         if (null !== $accessToken) {
             $query = 'https://graph.facebook.com/v2.2/?id='.urlencode($url).'&'.$accessToken;
         } else {
-            $query = 'https://graph.facebook.com/fql?q='
-                     .urlencode('SELECT total_count FROM link_stat WHERE url="'.$url.'"');
+            $query = 'https://graph.facebook.com/?id='.urlencode($url);
```

Comment:
Shouldn't there be a version number of the API too, like in line 27?
In general we could combine this into a single line like:
`$query = 'https://graph.facebook.com/v2.2/?id='.urlencode($url).($accessToken !== null ? '&'.$accessToken : '');`